pkgs/by-name/la/lavalink/package.nix

```
@@ -0,0 +1,55 @@
+{
+  lib,
+  stdenv,
+  makeWrapper,
+  jdk21,
+  jdk ? jdk21,
+  fetchurl,
+  nixosTests,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  name = "lavalink";
```

```suggestion
  pname = "lavalink";
```
name is generated based on pname and version